src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Security/Utils/ClaimsPrincipalUtils.cs

```diff
@@ -1,20 +1,72 @@
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using System.Security.Claims;
 using GovUk.Education.ExploreEducationStatistics.Admin.Security;
+using GovUk.Education.ExploreEducationStatistics.Common.Services;
+using IdentityModel;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Models.GlobalRoles;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Models.GlobalRoles.RoleNames;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Security.SecurityClaimTypes;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.CollectionUtils;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Security.Utils
 {
     public static class ClaimsPrincipalUtils
     {
+        public static ClaimsPrincipal AuthenticatedUser()
+        {
+            return CreateClaimsPrincipal(
+                Guid.NewGuid(), 
+                SecurityClaim(ApplicationAccessGranted));
+        }
+        
+        public static ClaimsPrincipal BauUser()
+        {
+            // Give the BAU User the "BAU User" role, plus every Claim from the SecurityClaimTypes enum.
+            var claims = 
+                ListOf(RoleClaim(RoleNames.BauUser))
+                .Concat(EnumUtil.GetEnumValues<SecurityClaimTypes>().Select(SecurityClaim));
+                
+            return CreateClaimsPrincipal(
+                Guid.NewGuid(),
+                claims.ToArray());
+        }
+        
+        public static ClaimsPrincipal AnalystUser()
+        {
+            return CreateClaimsPrincipal(
+                Guid.NewGuid(),
+                RoleClaim(Analyst),
+                SecurityClaim(ApplicationAccessGranted),
+                SecurityClaim(AnalystPagesAccessGranted),
+                SecurityClaim(PrereleasePagesAccessGranted),
+                SecurityClaim(CanViewPrereleaseContacts));
```

Comment:
Hey, yeah I did think about this but decided to try to keep it simple for now.  Figured it could get a little messy given things like the UsersAndRolesDbContext seed data provides explicit ids for each claim entry in the db.  Definitely doable in some for or other though.